controllers/profiles/getOtherProfile.js

```diff
@@ -53,15 +53,25 @@ module.exports = async (req, res) => {
 
     const getFollowerCountResult = getFollowerCount?.[0]?.[0]?.count_follower;
     const getFollowingCountResult = getFollowingCount?.[0]?.[0]?.count_following;
+    const isFollowingResult = isFollowing?.[0]?.length > 0;
 
     const copyUser = {...user.dataValues};
     delete copyUser.following;
     delete copyUser.follower;
-    copyUser.isSignedMessageEnabled = false;
+
+    if (req?.userId) {
+      copyUser.is_following = isFollowingResult;
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isFollowingResult` is being used to set the `is_following` property of `copyUser`. However, there's no null check for `isFollowingResult` before using it. This could potentially lead to unexpected behavior if `isFollowingResult` is null or undefined.
```diff
- const isFollowingResult = isFollowing?.[0]?.length > 0;
+ const isFollowingResult = isFollowing?.[0]?.length > 0 || false;

...

- copyUser.is_following = isFollowingResult;
+ copyUser.is_following = isFollowingResult || false;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->